spec/models/tslr_claim_spec.rb

```diff
@@ -168,6 +183,14 @@
     end
   end
 
+  describe "#student_loan_repayment_amount" do
+    it "saves with monetary characters removed" do
```

Comment:
I'm approving the PR, but spotted one more thing: this spec is a little off. The behaviour of the code isn't to do with saving, but is more specifically the cleanup that happens on assignment. e.g. something like: 

```ruby
claim = TslrClaim.new
claim.student_loan_repayment_amount = "£ 5,000.40"
expect(claim.student_loan_repayment_amount).to eql(5000.40)
```

feel free to correct and merge.

Reply:
I've corrected the spec 👍